### PR TITLE
[Blog] [Pulsar Summit] Add pulsar summit san francisco 2022 blog

### PR DESCRIPTION
**Pull request by Sherlock113**
### Motivation and Modifications

Add the Pulsar Summit San Francisco 2022 blog

Some preview images:
Screen Shot 2022-07-15 at 11 52 59
Screen Shot 2022-07-15 at 11 52 48
Screen Shot 2022-07-15 at 11 52 39

### CODE REVIEW

File: site2/website/blog/2022-07-15-pulsar-summit-san-francisco.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+author: Alice Bi, Karin Landers
+title: "Speakers and Sponsorship Prospectus Announced for Pulsar Summit San Francisco 2022"
+date: 2022-07-15
+---
+
```
Comment (dave2wave) [inline site2/website/blog/2022-07-15-pulsar-summit-san-francisco.md:6]: There needs to a link to the pulsar-summit.org website's event page, the speaker list, and the schedule.

**Review by dave2wave — CHANGES_REQUESTED** [review]
Please link to the pulsar-summit.org website somewhere in the blog post.

File: site2/website/blog/2022-07-15-pulsar-summit-san-francisco.md (hunk 1 above)
Comment (Sherlock113) [reply site2/website/blog/2022-07-15-pulsar-summit-san-francisco.md:6]: Thanks. I added these links to the text.

**Review by dave2wave — APPROVED** [review]
Those changes look good.

**Pull request merged**